llvm/lib/IR/RepoTicket.cpp

```diff
@@ -117,10 +117,10 @@ static GODigestState accumulateGOFinalDigestOrInitialDigestAndGetGODigestState(
 }
 
 // Update the GO's hash value by adding the hash of its dependents.
-template <typename Function>
+template <typename FunctionType>
 static bool updateDigestUseDependenciesAndContributions(
     const GlobalObject *GO, MD5 &GOHash, GOStateMap &Visited, GOInfoMap &GOIMap,
-    Function AccumulateGODigest) {
+    FunctionType AccumulateGODigest) {
```

Comment:
I’d prefer to avoid the Type suffix here. We already know from line 120 that it’s a type and this just adds additional noise.

Reply:
Undo the change, thanks.